tests/behat/bootstrap/DbTrait.php

```diff
@@ -13,8 +13,9 @@
     public static function iSaveTheApplicationStatusInto($status)
     {
         $sqlFile = self::getSnapshotPath($status);
-        @unlink($sqlFile);
-        exec('pg_dump ' . self::$dbName . " --clean --inserts | sed '/EXTENSION/d' > {$sqlFile}", $output, $return);
+        // manual data
+        exec('echo "SET client_min_messages TO WARNING; truncate dd_user, dd_team, casrec, client cascade;" > '.$sqlFile);
+        exec('pg_dump ' . self::$dbName . "  --data-only  --inserts --exclude-table='migrations' | sed '/EXTENSION/d' >> {$sqlFile}", $output, $return);
         if (!file_exists($sqlFile) || filesize($sqlFile) < 100) {
```

Comment:
could possibly remove this check for file exists to speed things up further as the filesize will still fail if doesnt exist

Reply:
in fact lose the entire if ?

Reply:
I'd not remove it, I'd trigger a warning if not existing although the eception would be thrown anyway. Also, takes a few ms to run that command, nothing compare to the 15 minutes to run the whole suite, and therefore I'd not even bother making the change and make sure I didn't alter the logic. Maybe we can do that on master next time  we change it

Reply:
Yes, fair enough, I dont think it would even get that far as the exec statement above would fail to write to the file.